### ISSUE TITLE
Synchronisation with Traewelling unreliable / "too many requests"

### ISSUE THREAD

**Issue by gufojoe** [derf/travelynx#141]
The synchronization from Traewelling to Travelynx has become a hit-and-miss affair. I reviewed the synchronization log and there are a lot of "HTTP 429 Too Many Requests" errors.

**Comment by DRSchlaubi** [comment]
This was an issue like two yrs ago; after I messaged @derf on Discord, it seemed fixed, however, it seems like it's back, although it seems to handle rate limits better now, as all check-ins seem to get imported

**Comment by dreirund** [comment]
I have this too:

---

![image](https://github.com/user-attachments/assets/b1793341-de11-43f0-9545-ecd4e14fe3c7)

---

![image](https://github.com/user-attachments/assets/18d9bfd0-eba4-4bc8-967d-2807607800d8)

---

Regards!

**Comment by Ein-Tim** [comment]
Syncing between travelynx and Traewelling is currently not supported, maybe it would make sense to close this issue and open a new one should this still be an issue once syncing works gain?

**Comment by derf** [comment]
yep. Sync will return one day, but until then this is sort of resolved.
